src/main/flight/gps_rescue.c

```diff
@@ -267,7 +267,7 @@ static void rescueAttainPosition(void)
     // D component is error based, so includes positive boost when climbing and negative boost on descent
     float verticalSpeed = ((altitudeError - previousAltitudeError) / rescueState.sensor.altitudeDataIntervalSeconds);
     previousAltitudeError = altitudeError;
-    verticalSpeed += rescueState.intent.descentRateModifier * verticalSpeed;
+    verticalSpeed += rescueState.intent.descentRateBooster * verticalSpeed;
```

Comment:
The current code adds the gain to D before smoothing by adding the boost on the verticalSpeed value before the filter.
Its important to add to D at the start of the descent because otherwise there is too much delay in the altitude response.
The modification proposed above removes the increase in D at the start, but I know from careful testing that it is very useful to boost D at this time, especially when the target descent rate is higher at the start from higher altitudes.

Reply:
The suggested code is equivalent to original one. Just reordering calculation and renaming descentRateBooster to throttleDGain.

Reply:
I've addressed this now, I think.